omero/sysadmins/unix/install-web.txt

```diff
@@ -131,17 +132,17 @@ OMERO.web maintenance
 
           $ bin/omero config set omero.web.caches '{"default": {"BACKEND": "django.core.cache.backends.memcached.MemcachedCache", "LOCATION": "127.0.0.1:11211", "TIMEOUT": "86400"}}'
 
-.. note::
-    The generated Nginx and Apache configuration files will automatically
-    display a maintenance page if an attempt is made to access OMERO.web whilst
-    it is not running.
-
 
 .. _customizing_your_omero_web_installation:
 
 Customizing your OMERO.web installation
 ---------------------------------------
 
+OMERO.web offers a number of configuration options.
+Depending on the deployment choice, OMERO.web will not activate
+configuration changes until either gunicorn is restarted using
```

Comment:
either? Or what? This is confusing. Not clear what '(nginx)' at the end means either